opensrp-reveal/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
         minSdkVersion 18
         targetSdkVersion 28
         versionCode 21
-        versionName "5.2.13"
+        versionName "5.3.0-RC1"
```

Comment:
Could we update the version from a release candidate

Reply:
done